asydata/chimeraData/R3BAsyChimeraPoint.cxx

```diff
@@ -0,0 +1,102 @@
+// -------------------------------------------------------------------------
+// -----                      R3BAsyChimeraPoint source file              -----
+// -------------------------------------------------------------------------
+
+#include "R3BAsyChimeraPoint.h"
+
+#include <iostream>
+
+using std::cout;
+using std::endl;
+using std::flush;
+
+// -----   Default constructor   -------------------------------------------
+R3BAsyChimeraPoint::R3BAsyChimeraPoint()
+    : FairMCPoint()
+{
+    fX_out = fY_out = fZ_out = 0.;
+    fPx_out = fPy_out = fPz_out = 0.;
+    fZFF = fAFF = 0.;
+}
+// -------------------------------------------------------------------------
+
+// -----   Standard constructor   ------------------------------------------
+R3BAsyChimeraPoint::R3BAsyChimeraPoint(Int_t trackID,
+                                 Int_t detID,
+                                 Int_t detCopyID,
+                                 Double_t Z,
+                                 Double_t A,
+                                 TVector3 posIn,
+                                 TVector3 posOut,
+                                 TVector3 momIn,
+                                 TVector3 momOut,
+                                 Double_t tof,
+                                 Double_t length,
+                                 Double_t eLoss,
+                                 Double_t lightYield,
+                                 Double_t slow)
+    : FairMCPoint(trackID, detID, posIn, momIn, tof, length, eLoss)
+{
+    fDetCopyID = detCopyID;
+    fX_out = posOut.X();
+    fY_out = posOut.Y();
+    fZ_out = posOut.Z();
+    fPx_out = momOut.Px();
+    fPy_out = momOut.Py();
+    fPz_out = momOut.Pz();
+    fZFF = Z;
+    fAFF = A;
+    fLightYield=lightYield;
+    fSlow= slow;
+}
+// -------------------------------------------------------------------------
+
+// -----   Destructor   ----------------------------------------------------
+R3BAsyChimeraPoint::~R3BAsyChimeraPoint() {}
+// -------------------------------------------------------------------------
+
+// -----   Public method Print   -------------------------------------------
+void R3BAsyChimeraPoint::Print(const Option_t* opt) const
```

Comment:
The print function is not used anymore, please, remove it. See for instance:
https://github.com/R3BRootGroup/R3BRoot/blob/dev/r3bdata/alpideData/R3BAlpidePoint.cxx
https://github.com/R3BRootGroup/R3BRoot/blob/dev/r3bdata/alpideData/R3BAlpidePoint.h